### DISCUSSION TITLE
White Earring Artifact

### DISCUSSION THREAD

**Discussion by thatnerdyaigirl** [hacksider/Deep-Live-Cam#326] [General]
We've observed a white earring stud appearing in some frames of female avatars, even though none of the people in the input data are wearing earrings. This artifact is not present in all frames, leading to further inconsistency.
![image](https://github.com/user-attachments/assets/a94dbadc-7267-4482-baf7-bfe390ab7c40)

**Comment by hacksider** [comment]
Model is not created by us ☺️
